docs/README.md

```diff
@@ -0,0 +1,114 @@
+
+`StrategyParams` define pairs of strategies and multipliers for the quorum:
+* Strategies refer to the `DelegationManager` in the EigenLayer core contracts, which tracks shares delegated to each operator for each supported strategy. Basically, a strategy is a wrapper around an underlying token - either an LST or Native ETH.
+* Multipliers determine the relative weight given to shares belonging to the corresponding strategy.
+
+When the `StakeRegistry` updates its view of an operator's stake for a given quorum, it queries the `DelegationManager` to get the operator's shares in each of the quorum's strategies and applies the multiplier to the returned share count.
+
+For more information on the `DelegationManager`, see the [EigenLayer core docs][core-docs-m2].
+
+##### Operator Sets and Churn
+
+Quorums define a maximum operator count as well as parameters that determine when a new operator can replace an existing operator when this max count is reached. The process of replacing an existing operator when the max count is reached is called "churn," and requires a signature from the Churn Approver.
+
+These definitions are contained in a quorum's `OperatorSetParam`, which the Owner can configure via the `RegistryCoordinator`. A quorum's `OperatorSetParam` defines both a max operator count, as well as stake thresholds that the incoming and existing operators need to meet to qualify for churn.
+
+*Additional context*:
+
+Currently for EigenDA, the max operator count is 200. This maximum exists because EigenDA requires that completed "jobs" validate a signature by the aggregate BLS pubkey of the operator set over some job parameters. Although an aggregate BLS pubkey's signature should have a fixed cost no matter the number of operators, it may be the case that not all operators sign off on a job.
+
+When this happens, EigenDA needs to provide a list of the pubkeys of the non-signers to subtract them out from the quorum's aggregate pubkey ("Apk"). The limit of 200 operators keeps the gas costs reasonable in a worst case scenario. See `BLSSignatureChecker.checkSignatures` for this part of the implementation.
+
+In order to prevent the operator set from getting calcified, the churn mechanism was introduced to allow operators to be replaced in some cases. Future work is being done to increase the max operator count and refine the churn mechanism.
+
+##### Hooking Into EigenLayer Core
+
+The main thing that links an AVS to the EigenLayer core contracts is that when EigenLayer operators register/deregister with an AVS, the AVS calls these functions in EigenLayer core:
+* [`DelegationManager.registerOperatorToAVS`][core-registerToAVS]
+* [`DelegationManager.deregisterOperatorFromAVS`][core-deregisterFromAVS]
+
+These methods ensure that the operator registering with the AVS is also registered as an operator in EigenLayer core. In this repo, these methods are called by the `RegistryCoordinator`.
```

Comment:
the other important check is that the registration is "consensual" -- we use a signature to ensure that the operator approves of the call from the AVS.
perhaps you want to add this, or it can go in the more detailed part?

Reply:
good note, but i'm keeping the top level concepts basic. i'll definitely include that in the deeper docs.